### ISSUE TITLE
update freedesktop runtime to 20.08

### ISSUE THREAD

**Issue by eszlari** [flathub/org.opencpn.OpenCPN#1]

**Comment by leamas** [comment]
This was discussed when this package was reviewed for inclusion early this autumn. The reason to update are obvious. OTOH, there is a plugin ecosystem, so a change in the runtime leads to an ABI break which needs to be handled in  all plugins.

There are three issues whch will lead to an ABI break: a flatpak runtime update, gtk2 -> gtk3 and an update of the wxWidgets framework 3.0 -> 3.2. Of these, wxWidgets 3.o is yet not available -- it is due Dec 15,

To summarize: A runtime update is indeed needed, but would IMHO be better off coordinated with the wxWidgets 3.0 release.

**Comment by lionirdeadman** [comment]
What is the status for this? It seems moving to wxWidgets 3.0 has been done and the beta branch uses 20.08

**Comment by leamas** [comment]
We have a upcoming upstream release 5.6.0 under way. This will support 20.08, hence the beta status.
